src/registry-client/src/index.ts

```diff
@@ -331,6 +332,8 @@ export class RegistryClient {
     url: URL | string,
     options: RegistryClientRequestOptions = {},
   ): Promise<CacheEntry> {
+    logRequest(url, 'start')
```

Comment:
I guess the other states are going to come later?

Reply:
yeah, this currently just counts how many requests are initiated